### PR TITLE
Yu frontend journal

### PR DESCRIPTION
**Pull request by yw3028**

### CODE REVIEW

**Review by jmaciaa — APPROVED** [review]
Great!

File: client/src/Components/JournalCard/JournalCard.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import moment from 'moment';
+
+import * as S from './JournalCardStyle';
+
+const JournalCard = ({ journal }) => {
+  return (
+    <S.JournalCard>
+      <S.JournalDate>
+        <div style={{padding:'5px'}}>{moment(journal.date).format('ddd DD').toUpperCase()}</div>
```
Comment (jaumefapa) [inline client/src/Components/JournalCard/JournalCard.jsx:10]: Remove html inline style

File: client/src/Components/JournalCard/JournalCardStyle.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/macro';
+
+export const JournalCard = styled.div`
+  width: 16rem;
+  height: auto;
+  margin: auto 40px auto 0;
+  box-shadow: 0 0 5px lightgray;
+  position: relative;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```
Comment (jaumefapa) [inline client/src/Components/JournalCard/JournalCardStyle.js:9]: I thought we were using 'Roboto'. We need consistency or create a theme and use variables

File: client/src/Pages/Journal/Journal.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    const key = moment(journal[date]).format('MMMM YYYY');
+    return {
+      ...result,
+      [key]: [...(result[key] || []), journal],
+    };
+  }, {});
+};
+
+const Journal = () => {
+  const [journals, setJournals] = useState([]);
```
Comment (jaumefapa) [inline client/src/Pages/Journal/Journal.jsx:19]: This state should be 'journal' and in my case I should talk about 'journalEntry'

**Review by jaumefapa — APPROVED** [review]

**Pull request closed without merging**